rconweb/api/views.py

```diff
@@ -82,25 +82,39 @@ def get_version(request):
 
 @csrf_exempt
 def public_info(request):
-    status = ctl.get_status()
+    gamestate = ctl.get_gamestate()
+    curr_players, max_players = tuple(map(int, ctl.get_slots().split("/")))
```

Comment:
We're pulling the total number of players on the server out of `get_slots` here but we also have that information in `get_gamestate`.

I think there's a chance that sometimes we could return inconsistent information since we're getting the server population through two different calls that might return different numbers if users have connected/disconnected between the calls?

Maybe just add together the total number of allied and axis players that we're getting through `get_gamestate`?

`get_slots` is on a 20 second cache timer also, right now `get_gamestate` isn't cached.

Reply:
This is indeed a good note and the cache timing may lead to inconsistencies.
However the number calculated from `get_gamestate` won't be accurate either since it does not include 'unassigned' players (players that have connected but not picked a team yet).

Reply:
Dang I can't believe I didn't think of that. Excellent point. Maybe we should have `get_gamestate` invalidate the cache for `get_slots`?

Reply:
Mhh this would add an obscure relationship between those two functions, which is not desirable.
Maybe we can just reduce the ttl *thinking* 5 seconds, and ignore the remaining edge cases. Since those will always be present at one point or another